cluster-autoscaler/cloudprovider/clusterapi/README.md

```diff
@@ -129,7 +129,16 @@ these environment variables to set the namespace to deploy into as well as the i
 
 ```
 export AUTOSCALER_NS=kube-system
-export CAPI_NS=capi-system
 export AUTOSCALER_IMAGE=us.gcr.io/k8s-artifacts-prod/autoscaling/cluster-autoscaler:v1.20.0
 envsubst < examples/deployment.yaml | kubectl apply -f-
 ```
+
+<aside class="note">
+
+<h1> Note </h1>
+
+The `cluster-autoscaler-management` role for accessing cluster api scalable resources is scoped to `ClusterRole`.
+This may not be ideal for all environments (eg. Multi tenant environments).
+In such cases, it is recommended to scope it to a `Role` mapped to a specific namespace.
+
+</aside>
```

Comment:
i would prefer not to use html here, i'm happy if you just make a level 3 header for the note, eg
```suggestion

### A note on permissions

The `cluster-autoscaler-management` role for accessing cluster api scalable resources is scoped to `ClusterRole`.
This may not be ideal for all environments (eg. Multi tenant environments).
In such cases, it is recommended to scope it to a `Role` mapped to a specific namespace.
```